main/zoe/api/zoe-contract-facet.md

```diff
@@ -166,23 +163,39 @@ Zoe.
 const { issuerKeywordRecord, keywords, terms } = zoe.getInstanceRecord()
 ```
 
-## zcf.getCurrentAllocation(offerHandle, sparseKeywords)
+## zcf.getCurrentAllocation(offerHandle, brandKeywordRecord)
 - `offerHandle` <router-link to="/glossary/#handle">`{Array <Handle>}`</router-link>
-- `sparseKeywords` sparseKeywords is an array of string keywords, which may be a subset of allKeywords.
-- Returns: <router-link to="/zoe/api/records.html#amount-keyword-record">`{<AmountKeywordRecord>}`</router-link>
+- `brandKeywordRecord` <router-link to="/zoe/api/records.html#brand-keyword-record">
+`{[<BrandKeywordRecord>]}`</router-link> is an optional record mapping keywords to brands for each of the offers.
+- Returns: <router-link to="/zoe/api/records.html#amount-keyword-record">
+`{<AmountKeywordRecord>}`</router-link>
+
+Get the amounts currently allocated to the offer for the keywords specified by brandKeywordRecord. If
+brandKeywordRecord is not specified, amounts are returned for only the Keywords that currently have
+allocated amounts (including empty amounts.) This might mean that amounts that will be included in the
+payout could be omitted if not specified.
 
-Get the amounts associated with the sparseKeywords for the offer.
+If brandKeywordRecord is specified and amounts for some keywords haven't been assigned to the
```

Comment:
Change to:  'If `brandKeywordRecord` is specified, empty amounts are filled in for keywords without amounts assigned to the offer.'